Makefile

```diff
@@ -444,7 +460,7 @@ $(GOPATH)/bin/goreman:
 
 .PHONY: start
 ifeq ($(RUN_MODE),local)
-start: install executor-image controller cli $(GOPATH)/bin/goreman
+start: install controller cli $(GOPATH)/bin/goreman
```

Comment:
this change means that we don't wait for the executor to be built before starting up, useful for UI only work